### PR TITLE
es: Update browser compat/spec sections (part 1)

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates some of the browser compatibility data and specification sections for the Spanish locale.  Part of work for #11594.

### CODE REVIEW

**Comment by queengooborg** [comment]
Ah, didn't realize "con" was preferred over "de" here, I'll keep that in mind for future PRs!

File: files/es/web/javascript/reference/global_objects/error/filename/index.md [hunk 1]
```diff
@@ -24,11 +24,11 @@ throw e;
 
 ## Especificaciones
 
-No forma parte de ningún estándar.
+{{Specifications}}
```
Comment (Graywolf9) [inline files/es/web/javascript/reference/global_objects/error/filename/index.md:27]: As `en-US`

```suggestion
No forma parte de ningún estándar.
```

File: files/es/web/javascript/reference/global_objects/internalerror/index.md [hunk 2]
```diff
@@ -69,11 +69,11 @@ Para obtener más información, consulta {{JSxRef("Errors/Too_much_recursion", "
 
 ## Especificaciones
 
-No forma parte de ningún estándar.
+{{Specifications}}
```
Comment (Graywolf9) [inline files/es/web/javascript/reference/global_objects/internalerror/index.md:72]: As `en-US`

```suggestion
No forma parte de ningún estándar.
```

File: files/es/web/javascript/reference/global_objects/internalerror/internalerror/index.md [hunk 3]
```diff
@@ -33,11 +33,11 @@ new InternalError("Fallo del motor");
 
 ## Especificaciones
 
-No forma parte de ningún estándar.
+{{Specifications}}
```
Comment (Graywolf9) [inline files/es/web/javascript/reference/global_objects/internalerror/internalerror/index.md:36]: As `en-US`

```suggestion
No forma parte de ningún estándar.
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much for your work and attentions!

**Pull request merged**